x-pack/plugins/uptime/server/graphql/pings/schema.gql.ts

```diff
@@ -11,6 +11,11 @@ export const pingsSchema = gql`
     query: Query
   }
 
+  type PingResults {
+    total: UnsignedInteger!
+    pings: [Ping!]
```

Comment:
Sorry, missed this when we were pairing. If you make the edit below and re-run the `infer...` script this should be golden.